diff --git a/node_modules/sponsorkit/.DS_Store b/node_modules/sponsorkit/.DS_Store
new file mode 100644
index 0000000..60bd31c
Binary files /dev/null and b/node_modules/sponsorkit/.DS_Store differ
diff --git a/node_modules/sponsorkit/dist/shared/sponsorkit.D7ijwnCH.mjs b/node_modules/sponsorkit/dist/shared/sponsorkit.D7ijwnCH.mjs
index a55b2f8..c84d0df 100644
--- a/node_modules/sponsorkit/dist/shared/sponsorkit.D7ijwnCH.mjs
+++ b/node_modules/sponsorkit/dist/shared/sponsorkit.D7ijwnCH.mjs
@@ -907,7 +907,7 @@ ${JSON.stringify(data.errors, null, 2)}`);
 function makeQuery(login, type, activeOnly = true, cursor) {
   return graphql$1`{
   ${type}(login: "${login}") {
-    sponsorshipsAsMaintainer(activeOnly: ${Boolean(activeOnly)}, first: 100${cursor ? ` after: "${cursor}"` : ""}) {
+    sponsorshipsAsMaintainer(activeOnly: ${Boolean(activeOnly)}, includePrivate: true, first: 100${cursor ? ` after: "${cursor}"` : ""}) {
       totalCount
       pageInfo {
         endCursor
@@ -1109,7 +1109,7 @@ function createSponsorFromOrder(order) {
       type: getAccountType(order.fromAccount.type),
       login: slug,
       avatarUrl: order.fromAccount.imageUrl,
-      websiteUrl: normalizeUrl(getBestUrl(order.fromAccount.socialLinks)),
+      websiteUrl: normalizeUrl(getBestUrl(order.fromAccount.socialLinks) || order.fromAccount.website),
       linkUrl: `https://opencollective.com/${slug}`,
       socialLogins: getSocialLogins(order.fromAccount.socialLinks, slug)
     },
@@ -1208,6 +1208,7 @@ function makeTransactionsQuery(id, slug, githubHandle, offset, dateFrom, dateTo)
             }
             isIncognito
             imageUrl(height: 460, format: png)
+            website
           }
         }
       }
@@ -1245,6 +1246,7 @@ function makeSubscriptionsQuery(id, slug, githubHandle, offset, activeOnly) {
             }
             isIncognito
             imageUrl(height: 460, format: png)
+            website
           }
         }
       }
